bin/pr-update

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
+usage="
+$0: Updates PR containers made with pr-review.sh.
```

Comment:
add: Pulls latest updates from github and runs bin/drupal-update on the PR cms container.